### PR TITLE
Frontend fixes

### PR DESCRIPTION
**Pull request by ashishrawat0**
1. Value changed to Employee Name.
2. Changed Credit card to Credit Card.
3. Fixed dialog size.
4. Removed unnecessary code.
5. Input for customer search on the main screen.
6. Added search icon to phone
7. Added ngxMask for phone number

### CODE REVIEW

File: frontend/uv-poc/src/app/customer-details/customer-details.component.html [hunk 1]
```diff
@@ -8,6 +8,13 @@
       <th mat-header-cell *matHeaderCellDef class="text-right">Phone number</th>
       <td mat-cell *matCellDef="let details" class="text-right">
         {{ details.phoneNo }}
+        <mat-icon
+          aria-hidden="false"
+          class="icon-display"
+          aria-label="Example home icon"
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/customer-details/customer-details.component.html]: This isn't exactly what we want to see when hovering over the icon `Example home icon`
Please come up with something better

File: frontend/uv-poc/src/app/home/home.component.ts [hunk 2]
```diff
@@ -35,4 +41,14 @@ export class HomeComponent implements OnInit {
       }
     );
   }
+  openDialog(): void {
+    this.check = true;
+    var phoneNo = this.phoneNoForm.value.phoneNo;
+    if (phoneNo != '') {
+      const dialogRef = this.dialog.open(CustomerHistoryComponent, {
+        width: '800px',
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/home/home.component.ts]: The same dialog opens with 600px in `customer-details.component.ts`

File: frontend/uv-poc/src/app/order/order.component.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ export class OrderComponent implements OnInit {
   transactionId;
   errorMessage;
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/order/order.component.ts]: Is this being used at all?

File: frontend/uv-poc/src/app/transaction-detail/transaction-detail.component.html [hunk 4]
```diff
@@ -1,12 +1,15 @@
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="#">Website name</a>
+  <a class="navbar-brand"
+    >Details for transaction id:
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/transaction-detail/transaction-detail.component.html]: Rename `Details for transaction id:` to `Transaction`

File: frontend/uv-poc/src/app/transaction-detail/transaction-detail.component.html [hunk 5]
```diff
@@ -103,39 +104,39 @@
...
       <mat-tab label="Consultant">
-        <app-consultant-details></app-consultant-details>
+        <app-consultant-details [transactionId]="transactionId"></app-consultant-details>
       </mat-tab>
 
       <mat-tab label="Order">
         <app-order></app-order>
       </mat-tab>
-      <mat-tab label="Credit card">
+      <mat-tab label="Credit card" style="margin-bottom: 0; overflow: hidden;">
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/transaction-detail/transaction-detail.component.html]: Rename `Credit card` to `Credit Card`

**Review by KartikyaThakur — CHANGES_REQUESTED** [review]

File: frontend/uv-poc/src/app/order/order.component.ts (hunk 3 above)
Comment (ashishrawat0) [reply frontend/uv-poc/src/app/order/order.component.ts]: This was lakshya component but i will still look into it

File: frontend/uv-poc/src/app/home/home.component.ts (hunk 2 above)
Comment (ashishrawat0) [reply frontend/uv-poc/src/app/home/home.component.ts]: Changed to 600px

**Pull request merged**